index.html

```diff
@@ -2092,319 +1905,526 @@ <h1>Core Representations</h1>
-            </h2>
-
-            <p>
-The <a>DID document</a> is serialized following the rules in the JSON processor,
-with one addition: <a>DID documents</a> MUST include the <code>@context</code>
-property.
-            </p>
-
-            <dl>
-                <dt>@context</dt>
-                <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
-<code>@context</code> property MUST exist be in the DID properties extension
-registry.
-                </dd>
-            </dl>
-
-            <p class="issue" data-number="202">
-This specification defines globally interoperable documents, and the requirement
-that the context value be in the DID registry means that different JSON-LD
-processors can consume the document without having to dereference anything in
-the context field. However, a pair of producers and consumers can have local
-extension agreements. This needs to be clarified in a section on extensibility
-and linked here when available.
-            </p>
-
-        </section>
-
-        <section>
-            <h2>
-Consumption
-            </h2>
-
-            <p>
-The top-level element MUST be a JSON object. Any other data type at the top
-level is an error and MUST be rejected. This top-level JSON object is
-interpreted using JSON-LD processing under the rules of the defined
-<code>@context</code> fields.
-            </p>
-
-            <dl>
-                <dt>@context</dt>
-                <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. If more than one <a>URI</a> is
-provided, the <a>URIs</a> MUST be interpreted as an ordered set. It is
-RECOMMENDED that dereferencing each <a>URI</a> results in a document containing
-machine-readable information about the context.
-                </dd>
-            </dl>
-
-
-            <p>
-Unknown object member names MUST be ignored as unknown properties.
-            </p>
-
-            <p class="issue" data-number="205">
-This specification needs to define clear and consistent rules for how to handle
-unknown data members on consumption, and this section needs to be updated with
-that decision.
-            </p>
-
-        </section>
+  <section>
+    <h4>Production</h4>
+<p>  When producing DID Documents that are represented as CBOR, in addition to the suggestions in section 3.9 of the CBOR [[RFC7049]] specification for deterministic mappings the following constraints of the DID Document model MUST be followed: </p>
+
+  <ul>
+  <li>Map keys MUST be strings.</li>  
+  <li>Integer encoding MUST be as short as possible.</li> 
+  <li>The expression of lengths in CBOR major types 2 through 5 MUST be as short as possible.</li>
+  <li>All floating point values MUST be encoded as 64-bits, even for integral values.</li> 
```

Comment:
This implies that CBOR inherits the JSON rules above, but doesn't state it directly. Is this the case?

Reply:
Perhaps it should say map keys should be `tstr` to be specific to CBOR?

Reply:
I was following the Canonical CBOR constraints from (section 3.9 of ietf 7049)[https://tools.ietf.org/html/rfc7049#section-3.9].  I suppose the mapped keys isn't in that section and I can move that to the dagCBOR section, but I thought it belonged here as be more in-line with the JSON and JSON-LD representations.  So this is a constraint to conform to the core data model.